CMakeLists.txt

```diff
@@ -20,59 +20,48 @@ set(CMAKE_CXX_EXTENSIONS OFF) #...without compiler extensions like gnu++11
 option(CMAKE_VERBOSE_MAKEFILE "Generate verbose Makefiles" OFF)
 include(GNUInstallDirs)
 
+# allow users to directly point to a kokkos install
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-find_package(Kokkos REQUIRED)
-
-option(Cabana_ENABLE_Serial "Build Cabana with Serial support" ON)
+set(Kokkos_INSTALL_PATH "" CACHE PATH "Kokkos installation location" )
+if(Kokkos_INSTALL_PATH)
+  find_package(Kokkos 3 PATHS ${Kokkos_INSTALL_PATH} NO_DEFAULT_PATH)
```

Comment:
This change is not advertised in your description.  Why do we want this?

Reply:
oh sorry will document - a user requested this so they could specify a Kokkos install outside of the prefix path in case they had multiple Kokkos variants installed. this would allow them to select the correct install in that case

Reply:
I would argue the user can prepend to the CMake search path and that option is not absolutely necessary.

Reply:
I have pinged the user who made the request to comment here on their needs

Reply:
Yeah, I would just use `CMAKE_PREFIX_PATH` for that. Also, isn't there `Kokkos_ROOT` exactly for that purpose?

Reply:
Agreed - I am removing this